2200077/문제4.py

```diff
@@ -0,0 +1,14 @@
+# 불균등 소득
+
+T = int(input())
+
+for t in range(T):
+    test_case = int(input())
+    income = list(map(int, input().split()))
+    average = sum(income) // test_case
```

Comment:
```suggestion
    average = sum(income) / test_case
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄